managed-databases/postgresql-and-mysql/how-to/upgrade-version.mdx

```diff
@@ -14,36 +14,39 @@ categories:
   - postgresql-and-mysql
 ---
 
-If your Database Instance uses an End of Life engine version and is no longer maintained, we recommend that you upgrade your engine version.
+If your database engine is outdated, you can upgrade the version to the latest one anytime via the Scaleway console.
+
+<Message tyepe="note>
+  We recommend that you upgrade your engine version if your Database Instance uses an End of Life engine version that is no longer maintained.
+</Message>
+
+  <Message type="important">
+    The version upgrade is only available for:
+      - Database Instances using outdated engine versions.
```

Comment:
the major version upgrade is available enven if the engine version is not end of life. I don't know if `outdated` is the right word for it 😅 

i.e you can upgrade a PostgreSQL 13 to PostgreSQL 14 or 15 even if PostgreSQL 13 EOL is scheduled to `November 13, 2025`